profiles/targets/clang/packages

```diff
@@ -0,0 +1,9 @@
+# Copyright 1999-2016 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+*sys-devel/llvm
+*sys-devel/clang
+*sys-libs/libcxx
+*sys-libs/libcxxrt
+*sys-libs/llvm-libunwind
```

Comment:
And, missing packages.build.
It specifies the packages to include in stage 1.

If clang is not included in stage1, will fail to create stage2.

You can easily make it.

cd profiles/targets/clang
cp packages packages.build
sed -i 's:^*::g' packages.build